bin/docker/run

```diff
@@ -5,12 +5,36 @@
 # http://www.boost.org/LICENSE_1_0.txt)
 set -e
 
+# CLI handling
+
+pass_credentials_bintray=
+cmd=""
+
+while [ "$1" != "" ]; do
+    param="$1"
+
+    case $param in
+        --credentials-bintray)
+            pass_credentials_bintray=true
+            ;;
+        *)
+            cmd="$@"
+            break
+            ;;
+    esac
+
+    shift
+done
```

Comment:
I can't _remove_ pre-defined credentials this way which is the main change here. I don't mind removing `--credentials-github` if you want as that indeed is doable by `BEAVER_DOCKER_ARGS`